Cargo.toml

```diff
@@ -35,6 +34,11 @@ constant_time_eq = "0.1.5"
 rayon = { version = "1.2.1", optional = true }
 cfg-if = "0.1.10"
 
+# replace with crates.io version when available (probably 0.3.0 or 0.2.1)
+[dependencies.tinyvec]
+git = "https://github.com/Lokathor/tinyvec"
+features = ["nightly_const_generics", "alloc"]
```

Comment:
nightly const-generics feature necessary for its `Array` impls on larger sized collections, notably `MAX_DEPTH`.

Reply:
This can be easily fixed without nightly by asking @Lokathor to include 54 too. I did get the `const-generics` added for this.